build.gradle

```diff
@@ -1129,7 +1129,7 @@ publishing {
     repositories {
         maven {
```

Comment:
Need to rename the repo here too.  Artifactory -> Google Artifact Registry maybe?